src/main/java/formflow/library/FileController.java

```diff
@@ -181,4 +213,117 @@ RedirectView delete(
       return new RedirectView("/error");
     }
   }
+
+  /**
+   * @param httpSession  - The current httpsession
```

Comment:
I don't think Javadocs like the hyphen (-) and they should probably be removed. 

Also, across this file, can we call be consistent with `httpsession`, in comments, and call it `HttpSession`, after the Java object, or more generally `HTTP Session` (generically what it is)?